src/main/java/sirius/web/sass/Generator.java

```diff
@@ -172,11 +178,10 @@ protected InputStream resolveIntoStream(String sheet) throws IOException {
                 return null;
             }
         } else {
-            InputStream stream = getClass().getResourceAsStream((sheet.startsWith("/") ? "" : "/") + sheet);
-            if (stream == null) {
-                stream = getClass().getResourceAsStream((sheet.startsWith("/") ? "" : "/") + "_" + sheet);
-            }
-            return stream;
+            return resources.resolve((sheet.startsWith("/") ? "" : "/") + sheet)
```

Comment:
**Optional:** Don't think thats necessary anymore as its also done internally by `Resources::resolve(scopeId, uri)`

Reply:
Thx, thinned the code down